x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/types.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum CrawlerPolicies {
+  Allow = 'allow',
```

Comment:
Why the difference in capitalization here as compared to CrawlerRules?

Reply:
I've updated this in 331ab90